packages/landing/app/page.tsx

```diff
@@ -71,18 +73,99 @@ export default function Home() {
+            <HeadingText size={4} className="pb-10 md:pb-0">
+              Frequently asked questions
+            </HeadingText>
+            <div className="w-full max-w-lg space-y-4">
+              <QandA q="What is Stackly?">
+                Stackly is a simple non-custodial DCA app that makes it easy to
+                do recurring buys of any token.
+              </QandA>
+              <QandA q="How does Stackly work?">
+                When you stack a token, stackly creates a contract for you with
+                the funds and uses CoW protocol to place recurring orders
+                (stacks) at the frequency you choose.
+              </QandA>
+              <QandA q="What is DCA?">
+                DCA stands for Dollar-Cost Averaging, which is an investment
+                strategy used in the financial markets. DCA involves regularly
+                investing a fixed amount of money at predetermined intervals,
+                regardless of the {"asset's"} price.
+              </QandA>
+              <QandA q="Why one should do DCA?">
+                <ul className="px-4 pb-4 list-decimal">
+                  <li>Regular Investing.</li>
+                  <li>Reduced Market Timing Risk.</li>
+                  <li>Risk Mitigation.</li>
+                  <li> Psychological Benefits.</li>
+                  <li> Long-Term Investment.</li>
+                  <li> Automatic Investing.</li>
+                </ul>
+                <p className="text-lg font-medium text-em-med">
+                  {`While DCA has its advantages, it's
+                  important to note that it may not always yield the highest
+                  returns compared to lump-sum investing if the market
+                  experiences significant gains over a short period.`}
+                </p>
+                <p className="text-lg font-medium text-em-med">
+                  {`The choice between DCA and lump-sum investing depends on an individual's
+                  financial goals, risk tolerance, and market outlook.
+                  Ultimately, DCA is a strategy that can help individuals build
+                  wealth steadily over time while reducing the risks associated
+                  with market timing and volatility. However, like all
+                  investment strategies, it should be carefully considered in
+                  the context of one's financial situation and objectives.`}
+                </p>
+                <p className="text-lg font-medium text-em-med">
+                  {`Ultimately, DCA is a strategy that can help individuals build
+                  wealth steadily over time while reducing the risks associated
+                  with market timing and volatility. However, like all
+                  investment strategies, it should be carefully considered in
+                  the context of one's financial situation and objectives.`}
+                </p>
```

Comment:
Too much text here, right?

Reply:
I think it's good for now. 

We'll do text review on the polishing phase when we have the whole website.